### PR TITLE
Remove svg assets from the build/dist

### PR DESCRIPTION
**Pull request by huwshimi**
This work was started in #3833 but that PR was never merged.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- clean-all and build the gui
- load the gui and add some apps/relations
- check that the relation icons appear correctly
- open the df and check that the select boxes have the chevron in them

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added Prettier to the codebase to format `.scss` files.
 - Update to Xterm 3.8.0.
 - Browserify has been replaced with Webpack which also now builds the SCSS files.
+- Update the canvas to use the existing svg sprites.
+- Removed svg assets from the build/dist.
```
Comment (hatched) [inline CHANGELOG.md]: unused svg assets?

File: jujugui/static/gui/src/app/init/topology/service.js [hunk 2]
```diff
@@ -1546,11 +1546,6 @@ class ServiceModule {
    * @method createServiceNode
    */
   createServiceNode(node, self) {
-    var staticURL = self.topo.staticURL || '';
```
Comment (hatched) [inline jujugui/static/gui/src/app/init/topology/service.js]: on line 1786 there are some paths that reference a static path, should those be updated to use this staticURL?

**Comment by hatched** [comment]
QA Notes: 
When dragging the create-relation icon the circle that follows the cursor does not have the icon in it but it does in develop.

File: jujugui/static/gui/src/app/init/topology/service.js (hunk 2 above)
Comment (hatched) [reply jujugui/static/gui/src/app/init/topology/service.js]: Thanks!

**Comment by hatched** [comment]
Thanks @huwshimi

**Pull request merged**